### PR TITLE
Simple Pacman Easy program in python

### PR DESCRIPTION
**Pull request by Prachi-05**
Added a simple solution to Pacman Easy

### CODE REVIEW

**Review by girlscript-blr — CHANGES_REQUESTED** [review]
Here are the following changes need to be implemented
- Please send PR request for `girlscript-blr:solution` branch in place of girlscript-blr:master branch
- Please don't check in any other files outside your folder i.e. `Prachi-05_Prachi` Please remove the extra REAMDME.md files from Shopping Cart Problem, which is also checked in as it may lead to merging conflicts.
- Also add a README.md file or a .doc or .pdf file inside your folder i.e. `Prachi-05_Prachi` attaching screenshots of the output along with writing a few lines on the approach taken to solve the problem.

**Comment by Prachi-05** [comment]
Made the requested changes.

File: Easy/2. Pacman Game/solutions/Prachi-05_Prachi/Pacman-Easy.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        self.clear()
+        for i in maze:
+            for j in i:
+                print(j,end='')
+            print()
+        
+
+
+    def move(self,x,y):
+        if(aself.maze[self.pacmanx+x][self.pacmany+y]!='|' and self.maze[self.pacmanx+x][self.pacmany+y]!='-'):
```
Comment (girlscript-blr) [inline Easy/2. Pacman Game/solutions/Prachi-05_Prachi/Pacman-Easy.py:47]: it should be **self** in place of aself.

**Review by girlscript-blr — CHANGES_REQUESTED** [review]
I am seeing the existing files in the repo to have been deleted. If I try to merge your code, the same files in our solution branch will also get deleted.
Screenshot 2020-08-29 at 1 57 17 PM

To resolve these conflicts there are 2 approaches to resolve this conflict
- You can take a new fork from `girlscript-blr:master` branch, copy-paste your named folder i.e. `Prachi-05_Prachi` along with your Easy solution files inside your folder in `Easy/2. Pacman Game/solutions/` and send a new PR.
- You can add the deleted files back into your `PacmanEasy` branch and make it exactly the same as in our `girlscript-blr:master` branch.

You can try any of the approaches. if you get stuck or need our help feel free to let us know, we will be happy to help.

**Comment by girlscript-blr** [comment]
The PR is continued with new PR #77

**Pull request closed without merging**